src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -182,20 +187,57 @@ export class CalciteTipManager {
     this.nextTip();
   };
 
+  tipManagerKeyDownHandler = (event: KeyboardEvent): void => {
+    switch (event.key) {
+      case "ArrowUp":
+        event.preventDefault();
+      case "ArrowRight":
+        this.nextTip();
+        break;
+      case "ArrowDown":
+        event.preventDefault();
+      case "ArrowLeft":
+        this.previousTip();
+        break;
+      default:
+        break;
+    }
```

Comment:
Might be nice for `Home` and `End` to go to the first or last tip.

Reply:
Does nextTip() go to the first tip if its the last one?

Reply:
> Does nextTip() go to the first tip if its the last one?
Yes, no matter how it gets triggered it will revolve.

> Might be nice for Home and End to go to the first or last tip.
I can add this. Is there hesitation? @asangma @kat10140 thoughts?